scripts/env_setup.sh

```diff
@@ -11,6 +11,25 @@ getDependencies(){
     ./get_dependencies.sh
 }
 
+#Get user bot token and store in json file
+getBotToken(){
+	# Prompt user for their bot token
+	read -p "Enter your bot token: " token
+
+	# Create json object
+	json_data=$(jq -n --arg token "$token" \
+		'{token: $token}')
```

Comment:
This is some fancy syntax with `jq`, good stuff. We are going to need to make sure that the user has `jq` installed though so the script can run properly. I think the best solution would be to add it to the `getDependencies` function and then ensure that one runs first.